code-challenges.test.js

```diff
@@ -15,43 +18,111 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+describe("arrShuffle", () => {
+  it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    const result1 = arrShuffle(colors1)
+    expect(result1).toHaveLength(colors1.length - 1)
+    expect(colors1.slice(1)).toEqual(expect.arrayContaining(result1))
```

Comment:
I would review jest testing. Your expect statement should be performing a function call, which is how it's testing that your function is doing what it's supposed to be doing. The format is: 
expect -function call- to equal -expected result-